<launch>

  <node pkg="detect" type="detect" name="detect" output="screen">
      launch-prefix="xterm -e gdb --args"
      
        <param name="image_topic" value="/image_in" />
        <param name="iris_param" value="879.0" />
		<param name="tolerance" value="10" />
		<param name="H_min" value="120" />
		<param name="S_min" value="70" />
		<param name="V_min" value="120" />
		<param name="H_max" value="160" />
		<param name="S_max" value="180" />
		<param name="V_max" value="255" />
		<param name="zoom" value="1" />

      <remap from="/image_in/compressed" to="/image_raw/compressed"/>
  </node>
  
</launch>
